### PR TITLE
Run python-modernize on repo for Python2-Python3 compatibility

### PR DESCRIPTION
**Pull request by bmagyar**
Ran `python-modernize -wn .` on repo

### CODE REVIEW

**Comment by bmagyar** [comment]
@youtalk could you give this a look please?

**Comment by efernandez** [comment]
LGTM but I don't have to test. @bmagyar If you've tested it and all looks good, I can merge it.

File: key_teleop/scripts/key_teleop.py [hunk 1]
```diff
@@ -7,6 +7,7 @@
 # Authors:
 #   * Siegfried-A. Gevatter
 
+from __future__ import absolute_import
```
Comment (artivis) [inline key_teleop/scripts/key_teleop.py]: Is this really necessary?

File: joy_teleop/scripts/joy_teleop.py [hunk 2]
```diff
@@ -1,4 +1,5 @@
 #!/usr/bin/env python
+from __future__ import absolute_import
```
Comment (artivis) [inline joy_teleop/scripts/joy_teleop.py]: Is this really necessary?

File: mouse_teleop/scripts/mouse_teleop.py [hunk 3]
```diff
@@ -7,7 +7,8 @@
 # Authors:
 #   * Enrique Fernandez
 
-import Tkinter
+from __future__ import absolute_import
```
Comment (artivis) [inline mouse_teleop/scripts/mouse_teleop.py]: Is this really necessary?

File: joy_teleop/scripts/joy_teleop.py [hunk 4]
```diff
@@ -269,7 +271,7 @@ def get_service_type(self, service_name):
         return self.service_types[service_name]
 
     def update_actions(self, evt=None):
-        for name, cmd in self.command_list.iteritems():
+        for name, cmd in six.iteritems(self.command_list):
```
Comment (artivis) [inline joy_teleop/scripts/joy_teleop.py]: Is this PR meant to anticipate for Noetic? If so why using `six`? Wouldn't it be clearer to branch and use plain Python3?
Note that those small changes were already done for Dashing.
The same comment applies to `mouse_teleop.py`.

**Pull request closed without merging**